content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: '.endswith()'
+Description: 'Checks whether or not a string ends with a given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `.endswith()` method checks a value against a given string and returns `True` if the string ends with that value. Otherwise, it returns `False`.
+
+## Syntax
+
+```pseudo
+string.endswith(value, start, end)
+```
+
+Given a `string`, the `.endswith()` method can be used in the following way:
+
+- The required `value` argument, which is checked if it exists at the end of the `string`. It is also case-sensitive.
+
+- Optionally, the `value` can be tested for whether it is at the end of a particular substring within the `string` using the `start` and `end` index arguments.
```

Comment:
Still thinking that we don't need those back ticks for string because we're referring to it as some value's data type rather than a required code component. Wondering about everyone's opinion on this @Dusch4593, @KTom101

Reply:
> Still thinking that we don't need those back ticks for string because we're referring to it as some value's data type rather than a required code component. Wondering about everyone's opinion on this @Dusch4593, @KTom101

I agree that we do not need them, my bad for that will update the file.

Reply:
@yangc95 is it fine now?

Reply:
Perfecto 👌🏻 

Also, it's all good 😊 For context, we made a similar change in another entry. Just making sure it is consistent.

Reply:
Oh shoot, but can we place those back on the value, start, and end keywords?

Reply:
> Oh shoot, but can we place those back on the value, start, and end keywords? @Pandz18

Yeah sure, sorry bout that.

Reply:
> Perfecto 👌🏻
> 
> Also, it's all good 😊 For context, we made a similar change in another entry. Just making sure it is consistent.

I understand 🙂